### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by jkrems**

### CODE REVIEW

**Comment by jkrems** [comment]
@ageitgey Is this good to go?

File: api/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+The hook will initialize testium (if it didn't happen already)
+and apply a number of changes to the current test suite:
+
+* Intercept errors and automatically save screenshots.
+* Override the default timeouts using the [`mocha.timeout`](/config.html#mocha-timeout) setting.
+* Install an after hook to ensure everything is properly cleaned up.
+* Inject a `browser` property onto the test context.
+* Switch out the prototype of the [`browser` export](/api/#browser) to point to [`testium.browser`](/api/#testium-browser).
+
+The `options` are forwarder to [`getTestium`](/api/#gettestium-options)
```
Comment (ageitgey) [inline api/index.md]: `forwarded to`

File: api/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+#### `browser`
+
+A proxy to [`testium.browser`](/api/#testium-browser) once the before hook finished.
```
Comment (ageitgey) [inline api/index.md]: Would read more clearly as:

After `beforeHook()` has been executed, this object will be a proxy to [`testium.browser`](/api/#testium-browser).

File: api/sync/index.md [hunk 3]
```diff
@@ -0,0 +1,637 @@
...
+#### `browser.clearCookie(name)`
+
+Delete a cookie by `name` that is visible to the current page.
+
+### Alerts & Confirms
+
+This API allows you to interact with
+alert, confirm, and prompt dialogs.
+
+Some browsers, notable phantomjs,
```
Comment (ageitgey) [inline api/sync/index.md]: 'notably phantomjs'

File: config.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+Testium uses [`rc`](https://www.npmjs.com/package/rc) for loading configuration.
+Be aware that command line arguments won't work
+since most test runners (e.g. `mocha`) have their own argv handling.
+
+### Top-Level Options
+
+#### `browser`
+
+The name of the browser to use.
+Expects a valid webdriver `browserName`.
```
Comment (ageitgey) [inline config.md]: Might be cool to say or link to info on how to get a list of browserNames supported by your installation.

File: config.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+If no browser is specified, testium will assume it's `'phantomjs'`.
+
+#### `desiredCapabilities`
+
```
Comment (ageitgey) [inline config.md]: "Additional desired webdriver capabilities" doesn't explain what this does. It's like writing a dictionary definition of a word by restating the word.

What does this option actually do? Why would I want to use it?

**Comment by ageitgey** [comment]
This looks great as a starting point.

I made a few suggestions for small improvements to the content itself, but none of them I would consider a blocker.

:shipit:

File: config.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+since most test runners (e.g. `mocha`) have their own argv handling.
+
+### Top-Level Options
+
+#### `browser`
+
+The name of the browser to use.
+If no browser is specified, testium will assume it's `'phantomjs'`.
+Typical values are `phantomjs`, `chrome`, or `firefox`.
+The latter two depend on the browsers being installed on your computer.
```
Comment (ageitgey) [inline config.md]: "The latter two depend on which browsers are installed on your computer."

**Comment by ageitgey** [comment]
One last nitpick and then :shipit:

**Pull request merged**